scss/inc/_select2.scss

```diff
@@ -49,7 +49,7 @@
 }
 
 .select2-container .select2-choice {
-    display: block;
+    display: flex;
```

Comment:
Have you tried to add `position: relative` to this selector?
The revert of line 56 should fix it https://github.com/oat-sa/tao-core-ui-fe/commit/8e999ca987b194f0e8398b4e3766a48926676268#diff-9ddf7e56610168b3d70cd82d877989af

Reply:
I try but then create the bug that I fix time ago about scroll

https://github.com/oat-sa/tao-core-ui-fe/pull/165